tests/evaluator-tests.js

```diff
@@ -21,44 +21,44 @@ describe('Evaluator', () => {
       arr: ['zero', 'one']
     })
 
-    it("return entire map when passed `[]`", () => {
+    it('return entire map when passed `[]`', () => {
       var result = evaluator.evaluate(state, [])
       expect(Immutable.is(state, result)).toBe(true)
     })
 
-    it("['top']", () => {
+    it('[\'top\']', () => {
```

Comment:
I don't know how esLint works but jshint will just work with consistency in your project. I think you have to choose one over the other so if you wanted double over single you would do 

```js
["double", "avoid-escape"]
```

but I prefer single